previousnext/k8s-skpr-solr
# WIP - CircleCI linting

#### What does this PR do?

Adds `make lint` command to CircleCI build.

Currently there are a lot of lint violations - I am leaving this PR in "WIP" state until they are resolved.
#### How should this be manually tested?

Ensure latest build of this branch passes in [CircleCI](https://circleci.com/gh/previousnext/k8s-skpr-solr/tree/circleci-lint)

#### Any background context you want to provide?

TBC

#### What are the relevant tickets?

TBC
